devops/jobs/AppWatcher.groovy

```diff
@@ -61,56 +59,63 @@ class AppWatcher {
 
         // for each application
         extraVars.get("APPS_TO_CONFIG").each { app_name, app_config ->
-            dslFactory.job(extraVars.get("FOLDER_NAME", "DockerCI") + "/" + app_name) {
 
-                def app_repo = app_config.get("app_repo", '')
-                def app_repo_branch = app_config.get('app_repo_branch', 'master')
+            def app_repo = app_config.get("app_repo", '')
 
-                logRotator common_logrotator
+            // we don't need a watcher job if there is no repository to watch
+            if (app_repo) {
```

Comment:
I don't want to create watcher jobs for applications without application code repositories (e.g. firefox, chrome, rabbitmq, xenial-common).